Documentation/docs-site/content/tutorial/access-secret-cfgmap-in-function.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Access secret/configmap in function"
+draft: false
+weight: 41
+---
+
+From fission v0.5.0 and later, functions are able to access [Secret](https://kubernetes.io/docs/concepts/configuration/secret/) and [ConfigMap](https://kubernetes.io/docs/concepts/storage/volumes/#configmap) specified by users.
+
+### Create Secret and ConfigMap
+
+You can create Secret and ConfigMap with CLI.
+
+``` bash
+$ kubectl -n default create secret generic foo --from-literal=TEST_KEY="TESTVALUE"
+$ kubectl -n default create configmap bar --from-literal=TEST_KEY=TESTVALUE
+```
+
+Or create both of them with following YAML.
+
+``` yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: default
+  name: foo
+data:
+  TEST_KEY: VEVTVFZBTFVF # value after base64 encode
```

Comment:
Is this valid base64?  echo TEST_VALUE | base64 is `VEVTVF9WQUxVRQ==`

Reply:
Here the original value is `TESTVALUE`, so base64(TESTVALUE) = VEVTVFZBTFVF